### ISSUE TITLE
Notifications fires multiple times. then web app opened in multiple tabs in each tab

### ISSUE THREAD

**Issue by failhunterby** [pushy/pushy-sdk-web#5]
Is it possible to check scope for each Pushy Instances?

**Comment by pushy** [comment]
Hi @failhunterby,
Thanks for reaching out. We'd be glad to assist.

If the notification arrives multiple times, please ensure you aren't sending to multiple Pushy device tokens belonging to the same user.

**Comment by failhunterby** [comment]
So. it was an scope issue, ill checked your source code, find serviceWorkerScope option, set is as "/" and it works. TY

**Comment by pushy** [comment]
Hi @failhunterby,
Thanks so much for letting us know how you managed to resolve the issue.

We've just released version `1.0.11` of the [Pushy Web SDK](https://pushy.me/docs/web-push/get-sdk) which changes the default service worker scope from `./` to `/`. This effectively works around the issue you encountered with multiple device registrations when calling `Pushy.register()` from multiple subdirectories.

If you're using JavaScript modules, install version 1.0.11 of the [Pushy Web SDK](https://github.com/pushy/pushy-sdk-web) from [npm](https://www.npmjs.com/package/pushy-sdk-web) using the following command:
```
npm install pushy-sdk-web@latest --save
```

Alternatively, import version `1.0.11` by adding the following line to the <head> tag:
```
<script src="https://sdk.pushy.me/web/1.0.11/pushy-sdk.js"></script>
```

Please let us know if there's anything else we can help with.
